Compilation Error With GCC

**Describe the bug**
I am trying to upgrade from Filament 1.9.19 to 1.15.1 and encountered a compilation with gcc 7.5 on Ubuntu 18.04. It appears that around [this](https://github.com/google/filament/commit/46e3d10e6264d2af987741d3e1945e04d3254df4#diff-766e1f0ff3e9d2aed6761f6809ff7e70a3aa0f2d3f08bf880bac7e45e83d1978) commit the namespace qualifications for a number of `using` declarations were removed. GCC complains about quite a few of the now unqualified `using` declarations. For example:
```
filament/include/backend/DriverEnums.h:810:36: error: declaration of ‘using CullingMode = enum class filament::backend::CullingMode’ [-fpermissive]
     using CullingMode = CullingMode;
                                    ^
filament/include/backend/DriverEnums.h:295:12: error: changes meaning of ‘CullingMode’ from ‘enum class filament::backend::CullingMode’ [-fpermissive]
 enum class CullingMode : uint8_t {
            ^~~~~~~~~~~
filament/include/backend/DriverEnums.h:812:40: error: declaration of ‘using BlendEquation = enum class filament::backend::BlendEquation’ [-fpermissive]
     using BlendEquation = BlendEquation;
                                        ^
filament/include/backend/DriverEnums.h:758:12: error: changes meaning of ‘BlendEquation’ from ‘enum class filament::backend::BlendEquation’ [-fpermissive]
 enum class BlendEquation : uint8_t {
            ^~~~~~~~~~~~~
filament/include/backend/DriverEnums.h:813:40: error: declaration of ‘using BlendFunction = enum class filament::backend::BlendFunction’ [-fpermissive]
     using BlendFunction = BlendFunction;
                                        ^
filament/include/backend/DriverEnums.h:767:12: error: changes meaning of ‘BlendFunction’ from ‘enum class filament::backend::BlendFunction’ [-fpermissive]
 enum class BlendFunction : uint8_t {
```
There is some discussion of that compiler error [here](https://stackoverflow.com/questions/41515989/may-i-declare-a-member-type-alias-to-a-type-in-a-surrounding-scope-using-the-sa) and [here](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1697r0.html).

The simple solution would be to add back in the qualifications necessary to satisfy gcc (e.g., `using CullingMode = backend::CullingMode`). I would be happy to submit a PR to fix this issue if you would accept it. I don't know the original motivation for removing the namespace qualifications.

**To Reproduce**
The following minimal program demonstrates the issue:
```
#include <filament/Engine.h>

int main() {
    return 0;
}
```
when compiled with:
`g++ -O2 -g -Ifilament/include main.cc`

**Expected behavior**
Program that includes Filament headers compiles successfully with gcc 7.5.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - GPU: NVIDIA Geforce RTX2060
 - Backend: OpenGL

--- a/filament/backend/include/backend/DriverEnums.h
+++ b/filament/backend/include/backend/DriverEnums.h
@@ -809,10 +809,10 @@ using AttributeArray = std::array<Attribute, MAX_VERTEX_ATTRIBUTE_COUNT>;
 //! Raster state descriptor
 struct RasterState {
 
-    using CullingMode = CullingMode;
-    using DepthFunc = SamplerCompareFunc;
-    using BlendEquation = BlendEquation;
-    using BlendFunction = BlendFunction;
+    using CullingMode = backend::CullingMode;
+    using DepthFunc = backend::SamplerCompareFunc;
+    using BlendEquation = backend::BlendEquation;
+    using BlendFunction = backend::BlendFunction;
 
     RasterState() noexcept { // NOLINT
         static_assert(sizeof(RasterState) == sizeof(uint32_t),

